platformcomponents/desktop/messages.json

```diff
@@ -72,6 +72,36 @@
           "text": "@theme-text-warning-normal"
         }
       }
+    },
+    "badge": {
+      "regular": {
```

Comment:
This is called 'flags' according to figma. To check this, click on the component and check the feature name the design team gave to it 
![image](https://user-images.githubusercontent.com/56999622/192314965-fc0c87c3-94ea-41e3-9c4b-7ec79cb8ab9c.png)